docs/concepts/jcloud/index.md

```diff
@@ -82,6 +84,36 @@ jina flow --uses flow.yml
 ```
 ````
 
+### Create a Secret or Job
+
+#### Create a Secret
+
+To create a Secret for a Flow:
+
+```bash
+jc create secret mysecret -f rich-husky-af14064067 --from-literal "{'env-name': 'secret-value'}"
+```
+
+```{tip}
+You can optionally pass the `--update` flag to automatically update the Flow spec with the updated secret information. This flag will update the Flow which is hosted on the cloud. Finally, you can also optionally pass a Flow's yaml file path with `--path` to update the yaml file locally.
```

Comment:
Link to the `env-from-secrets` parameter in Jina Docs. https://docs.jina.ai/cloud-nativeness/kubernetes/#deploy-flow-with-custom-environment-variables-and-secrets